History.md

```diff
@@ -48,6 +48,9 @@
   sensitive to non-reproducible factors in the external environment.
   https://github.com/meteor/meteor/pull/7668/commits/3313180a6ff33ee63602f7592a9506012029e919
 
+* The `meteor <command> ...` syntax will now work for any command
```

Comment:
See the [commit](https://github.com/meteor/meteor/pull/7668/commits/4300d261f3a265a8e29f06fdc9974ed2ee330cdf)! Essentially if you do `meteor npm install -g <package>` and that package installs a runnable script as `dev_bundle/bin/<command>`, then `meteor <command> ...` will work, similar to `meteor node` and `meteor npm`.

For example, you can run `yarn` independently from Meteor, but you probably want to be sure it's using the same version of `node` as your Meteor installation, in case you need to compile binary npm packages. You can verify that consistency by hand, but the easy way to get it right is to `meteor npm install -g yarn` and then use `meteor yarn ...` instead of `yarn`.